pkg/sdn/plugin/node.go

```diff
@@ -121,6 +162,36 @@ func (node *OsdnNode) Start() error {
 	return nil
 }
 
+// FIXME: this should eventually go into kubelet via a CNI UPDATE/CHANGE action
+// See https://github.com/containernetworking/cni/issues/89
+func (node *OsdnNode) UpdatePod(namespace string, name string, id kubeletTypes.ContainerID) error {
+	const pluginPath = "/opt/cni/bin/openshift-sdn"
+
+	cniConfig, err := getCNIConfig(node.masterKubeConfig, node.hostName, node.mtu)
+	if err != nil {
+		return err
+	}
+
+	args := &cniinvoke.Args{
+		Command:     "UPDATE",
+		ContainerID: id.String(),
+		NetNS:       "/blahblah/foobar", // plugin finds out namespace itself
```

Comment:
@smarterclayton CNI requires it to be set in https://github.com/containernetworking/cni/blob/master/pkg/skel/skel.go#L109 otherwise it will return an error when executing the plugin.  I could change our CNI plugin to not be based off CNI's skel.go, but that would be a lot more code.

The only reason we have this code in OpenShift anyway is because kubernetes doesn't have any kind of "UPDATE" method for network plugins; all it's got is setup/teardown and it's completely ignorant of anything that happens between those.